dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -87,7 +88,7 @@ public String getLatestVersionUrl() {
   public String getLatestIcgcGetVersionUrl(String os) {
     val result = getIcgcGetVersions();
     val latestVersion = result.stream().findFirst().orElse(null);
-    return getIcgcGetVersionUrl(latestVersion.get(), os);
+    return getIcgcGetVersionUrl(latestVersion.version, os);
```

Comment:
Can you please use `get()` instead? Best to always go through the public interface of a class. Thanks